test/test_util.py

```diff
@@ -7,15 +7,15 @@
 @pytest.mark.parametrize(
     "dpt_types,expected",
     [
-        (["DPT-1"], {"main": 1}),
-        (["DPT-1", "DPST-1-1"], {"main": 1, "sub": 1}),
-        (["DPT-7", "DPST-7-1"], {"main": 7, "sub": 1}),
-        (["DPST-5-1"], {"main": 5, "sub": 1}),
-        (["DPT-1", "DPT-5"], {"main": 5}),
-        (["DPT-14", "DPST-14-1"], {"main": 14, "sub": 1}),
-        (["DPST-6-10"], {"main": 6, "sub": 10}),
-        (["Wrong"], {}),
-        ([], {}),
+        ("DPT-1", {"main": 1, "sub": None}),
+        ("DPT-1 DPST-1-1", {"main": 1, "sub": 1}),
+        ("DPT-7 DPST-7-1", {"main": 7, "sub": 1}),
+        ("DPST-5-1", {"main": 5, "sub": 1}),
+        ("DPT-1 DPT-5", {"main": 5, "sub": None}),
+        ("DPT-14 DPST-14-1", {"main": 14, "sub": 1}),
```

Comment:
@marvin-w do you remember if you actually saw such in a project? I can not find any reliable documentation about that. My (test)projects seem to only have one of "DPT-*" or "DPST-*" in their "DatapointType" attributes.

Reply:
Never mind, I could reproduce this in ETS in CommObjects. Group addresses seem to have just one selectable.
I just don't know what to do with that information now 😬
```"DPT-4 DPST-4-1 DPST-4-2 DPT-5 DPST-5-1 DPST-5-3 DPST-5-4 DPST-5-5 DPST-5-6 DPST-5-10 DPST-5-100 DPT-6 DPST-6-1 DPST-6-10"```